projects/assessment-summary-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/AssessmentSubmitted.kt

```diff
@@ -49,11 +51,16 @@ class AssessmentSubmitted(
             domainEventService.publishEvents(regEvents)
             telemetryService.trackEvent("AssessmentSummarySuccess", telemetryParams)
         } catch (exception: Exception) {
-            telemetryService.trackEvent(
-                "AssessmentSummaryFailure",
-                telemetryParams + ("exception" to exception.message!!)
-            )
-            throw exception
+            when (exception) {
+                is NotFoundException -> throw IgnorableMessageException(exception.message!!, telemetryParams)
```

Comment:
Would it be better to be more explicit about which NotFoundExceptions we're ignoring?  i.e. update the Event and Person repositories to throw an IgnorableMessageException instead.

Also, maybe we should remove this try-catch altogether from here - all it's doing is adding a custom event for every exception, but we can see the exceptions in App Insights anyway so I don't think we need the extra event.